Source/Core/Core/PowerPC/MMU.cpp

```diff
@@ -1134,68 +1134,6 @@ TranslateResult JitCache_TranslateAddress(u32 address)
  * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
  */
 
-#define PPC_EXC_DSISR_PAGE (1 << 30)
-#define PPC_EXC_DSISR_PROT (1 << 27)
-#define PPC_EXC_DSISR_STORE (1 << 25)
-
-#define SDR1_HTABORG(v) (((v) >> 16) & 0xffff)
-#define SDR1_HTABMASK(v) ((v)&0x1ff)
-#define SDR1_PAGETABLE_BASE(v) ((v)&0xffff)
-#define SR_T (1 << 31)
-#define SR_Ks (1 << 30)
-#define SR_Kp (1 << 29)
-#define SR_N (1 << 28)
-#define SR_VSID(v) ((v)&0xffffff)
-#define SR_BUID(v) (((v) >> 20) & 0x1ff)
-#define SR_CNTRL_SPEC(v) ((v)&0xfffff)
-
-#define EA_SR(v) (((v) >> 28) & 0xf)
-#define EA_PageIndex(v) (((v) >> 12) & 0xffff)
-#define EA_Offset(v) ((v)&0xfff)
-#define EA_API(v) (((v) >> 22) & 0x3f)
-
-#define PA_RPN(v) (((v) >> 12) & 0xfffff)
-#define PA_Offset(v) ((v)&0xfff)
-
-#define PTE1_V (1 << 31)
-#define PTE1_VSID(v) (((v) >> 7) & 0xffffff)
-#define PTE1_H (1 << 6)
-#define PTE1_API(v) ((v)&0x3f)
-
-#define PTE2_RPN(v) ((v)&0xfffff000)
-#define PTE2_R (1 << 8)
-#define PTE2_C (1 << 7)
-#define PTE2_WIMG(v) (((v) >> 3) & 0xf)
-#define PTE2_PP(v) ((v)&3)
-
-// Hey! these duplicate a structure in Gekko.h
```

Comment:
This comment is still valid. `UReg_PTE` is duplicated by `UPTE_Lo` and `UPTE_Hi`. (Or is there a reason why we want to duplicate them in this way?)

Reply:
Technically we can get rid of UReg_PTE, since it's not used anywhere, if it's preferable to remove it then I can get rid of it. Given our current code, it's only ever operated on in terms of upper and lower halves.

Reply:
If it's not used, I think it would make sense to remove it.

Reply:
Added a commit that removes it